x11/auto-multiple-choice/Portfile

```diff
@@ -25,11 +25,12 @@ subport auto-multiple-choice-devel {}
 
 if {${subport} eq ${name}} {
     # release
-    set bitbucket_commit    "15d39fd2d4aa"
-    set amc_revision        "2132"
-    set amc_date            "2017111201428"
-    checksums               rmd160  325cce4d2c16b503f2de74922282188843c8131f \
-                            sha256  6c19ac832039b22ca266310c471aeff744c0e547d148e537969e6ba7668179dc
+    set revision            1
```

Comment:
Since the version increases when increasing the amc_revision, the revision line is not needed and should be deleted.